src/plugins/missing/README.md

```diff
@@ -0,0 +1,24 @@
+- infos = Information about the template plugin is in keys below
+- infos/author = Vid Leskovar <vid.leskovar5@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = missing
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = nodep
+- infos/metadata =
+- infos/description = A plugin which informs the user about a missing backend.
+
+## Introduction
+
```

Comment:
The name of the plugin already perfectly indicates the amount of documentation :laughing:

Reply:
I'm not even sure this plugin is used anymore. If it is used, it is very internal and the documentation should just be along the lines of: "internal, do not use"

Reply:
Yes, add "internal, do not use" for now. There used to be an internal plugin called "missing" that was mounted for plugins where the plugin loading failed so that "kdbGet" can be done successfully.